lang/python/python/Makefile

```diff
@@ -205,7 +205,7 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib $(1)/usr/lib/pkgconfig
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(2)/bin
```

Comment:
would it be an idea to move this to the `Host/Install` part ?
the thing with python[3] is that the Host/Install also installs some things that seem InstallDev-y;
and it may be useful when other packages do  PKG_BUILD_DEPDENDS := python/host

Reply:
I don't think this would work if moved to `Host/Install`, since this installs files from the target build (which may not exist yet during host build / install).

I think when other packages depend on host Python, they have Python code that needs to be run during build, but they aren't building modules or linking to host Python. (I guess there can be host packages that do that in the future.) I think keeping `Build/InstallDev` and `Host/Install` separate would be best.

Reply:
ack

Reply:
ack